src/platform/packages/shared/kbn-esql-validation-autocomplete/src/validation/validation.test.ts

```diff
@@ -454,9 +454,7 @@ describe('validation logic', () => {
           testErrorsAndWarnings(`row var = now() - 1 ${timeLiteral.name.toUpperCase()}`, []);
           testErrorsAndWarnings(`row var = now() - 1 ${capitalize(timeLiteral.name)}`, []);
           testErrorsAndWarnings(`row var = now() + 1 ${timeLiteral.name}`, []);
-          testErrorsAndWarnings(`row 1 ${timeLiteral.name} + 1 year`, [
-            `Argument of [+] must be [date], found value [1 year] type [duration]`,
-          ]);
+          testErrorsAndWarnings(`row 1 ${timeLiteral.name} + 1 year`, []);
```

Comment:
This is the validation check we are sacrificing but it is not an important one

Reply:
I also want to raise the importance of these tests. I don't think it is so important to test all these scenarios so possibly a refactoring of the validation tests while re-writing the validation makes sense @drewdaemon